CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 - Migrated `RegistrationConfirmPinScreen` to Mobius
 - Updated CI checks to look for [Clubhouse](https://app.clubhouse.io/simpledotorg) tickets instead of Pivotal Tracker
 - Migrated `RegistrationLocationPermissionScreen` to Mobius
+- [In Progress: 09 Jul 2020] Add support for `Assigned Facility ID`
```

Comment:
Does this need to be an `In Progress` update? It could just be `Added patient registration and assigned facility columns to the patient model`, right?

"Adding support for assigned facility" is not a single task, but a collection of multiple features and tasks which might span several releases. So we could split them up to be more granular.

Reply:
This makes sense